tasks/prepare-gitlab-runner.yml

```diff
@@ -8,6 +8,28 @@
   tags:
     - always
 
+- name: Install EPEL repo.
```

Comment:
Please, use dynamic include for OS-specific tasks

Reply:
Done